.github/workflows/dev.yml

```diff
@@ -49,9 +49,6 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust:
-          - version: "stable"
-            optional: false
```

Comment:
So this means we have no intent to get back to supporting several compiler versions? I thought this was fixable? I suppose it would require shipping several versions of the compiler in the compile-env, would that be an issue apart from the increased size?

Reply:
OK thank you